lms/djangoapps/certificates/queue.py

```diff
@@ -388,6 +391,20 @@ def add_cert(self, student, course_id, course=None, forced_grade=None, template_
             )
             return cert
 
+        if unverified:
+            cert.status = status.unverified
+            cert.save()
+            LOGGER.info(
+                (
+                    u"The user %s of course %s is not ID verified "
+                    u"but enrolled into verified course mode. "
+                    u"Set certificate status to unverified"
```

Comment:
Nitpick: Seeing this in a log would confuse me because it sounds like an instruction. I'd clean it up as follows:
> User %s has a verified enrollment in course %s but is missing ID verification. Certificate status has been set to `unverified`.